label_sync/main.go

```diff
@@ -513,7 +518,7 @@ type client interface {
 	GetRepoLabels(string, string) ([]github.Label, error)
 }
 
-func newClient(tokenPath, host string, dryRun bool) (client, error) {
+func newClient(tokenPath string, hosts []string, dryRun bool) (client, error) {
```

Comment:
I think this would be a little bit more convenient for some of the code if `hosts` was variadic / the last field.

Reply:
er, maybe this one as well, but I meant to search for and comment on NewClient in the github package. whoops